x-pack/plugins/security_solution/public/common/mock/endpoint/endpoint_alert_data_mock.ts

```diff
@@ -123,6 +123,20 @@ const generateEndpointAlertDetailsItemDataMock = (
       originalValue: ['windows'],
       isObjectArray: false,
     },
+    {
+      category: 'device',
+      field: 'device.id',
+      values: ['abfe4a35-d5b4-42a0-a539-bd054c791769'],
+      originalValue: ['abfe4a35-d5b4-42a0-a539-bd054c791769'],
+      isObjectArray: false,
+    },
```

Comment:
is this field "normal" or only present with Crowdstrike data?

If only applicable to Crowdstrike, then I suggest adding it to the crowd strike mock data method further below